qemu/tests/x86_cpu_flag_la57.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from virttest import cpu
+from virttest import utils_misc
+from virttest import env_process
+from virttest import error_context
+
+from provider.cpu_utils import check_cpu_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flag la57.
+    1) Check if current flags are in the supported lists on host, if no, cancel test
+    2) Otherwise, boot guest without the flag and without Icelake-Server model
+    3) Check cpu flags inside guest,shouldn't be there(only for linux guest)
+    4) Check the virtual address inside guest, should be '48 bits'
+    5) Boot guest with the flag
+    6) Check cpu flags inside guest,should be there(only for linux guest)
+    7) Check the virtual address inside guest, should be '57 bits'
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    flags = params["flags"]
+    check_host_flags = params.get_boolean("check_host_flags")
+    if check_host_flags:
+        check_cpu_flags(params, flags, test)
```

Comment:
This script is only for this single test case, I suppose here directly `check_cpu_flags(params, flags, test)` without the condition check would be fine.

Reply:
@vivianQizhu Thanks and update.